Modules/Journey/QuestDetailsFrame.lua

```diff
@@ -123,7 +145,7 @@ function _QuestieJourney:DrawQuestDetailsFrame(container, quest)
             local startQuests = {}
             local counter = 1
             for _, v in pairs(startNpc.questStarts) do
-                if (not v == quest.Id) then
+                if v ~= quest.Id then
```

Comment:
Nothing was ever rendered because of this. `not v` was being coerced to `false`

Reply:
oh wow, that is a great finding! 💪🏻